H2MLauncher.Core/ViewModels/ServerBrowserViewModel.cs

```diff
@@ -74,20 +80,33 @@ public ServerBrowserViewModel(
             CheckUpdateStatusCommand = new AsyncRelayCommand(CheckUpdateStatusAsync);
             CopyToClipBoardCommand = new RelayCommand<ServerViewModel>(DoCopyToClipBoardCommand);
             SaveServersCommand = new AsyncRelayCommand(SaveServersAsync);
+            OpenUpdatePageInBrowserCommand = new AsyncRelayCommand(DoOpenUpdatePageInBrowserCommand, () => _updateStatus != "" && _updateStatus != "Restart for new version!");
```

Comment:
The canExecute part is not necessary since the AsyncRelayCommand returns false when still executing

Reply:
It is, because when it is done, I don't want it to be able to execute again. As then it would redownload the new version and crash as it will attempt to change the (already downloaded exe) to the backup exe, but that one is still running.

Reply:
I see. Would be better to create a flag property then (like `AutoUpdateFinished`) this seems a little hacky.
Plus dont forget to annotate the property with `RaisesCanExecuteChangedFor`